### ISSUE TITLE
Changing The Quick Links Font To Match The Other Buttons

### ISSUE THREAD

**Issue by ZacharyDavidSaunders** [bamhm182/WGU-Plus#7]
After spending a few minutes digging through the HTML, I was able to find the font files that WGU uses for their accordion button titles. I'm able to download the font file and apply it to the WGU+ accordion "Quick Links" title.

Would you like the "Quick Links" accordion title font to match the other accordion titles? I can see cases both keeping it different (this helps to differentiate the extension's features from the stock WGU page options) and making it the same (makes the added element look more uniform).

Here's a before and after image:
![before and after](https://i.imgur.com/Y85yc1V.png "Before and after")

If you like the look of using the WGU font (the "After" option), as I do, let me know and I can create a pull request for this. Just let me know.

**Comment by bamhm182** [comment]
Good call. I think it looks better after as well. Pull it and I'll approve
it.

**Comment by ZacharyDavidSaunders** [comment]
Here's the PR: https://github.com/bamhm182/WGU-Plus/pull/8 

I realized that they use two fonts, "Lato" and "Futura WGU". In their CSS, they set Futura as the primary and Lato as a secondary. However, for some reason I see Lato when I view the site and as such I set Lato as our primary. It matches the other buttons for me, but you may want to double check on your end before accepting the PR.

**Comment by bamhm182** [comment]
Odd, for me, it looks like Futura is the font that matches the rest of the buttons. Not sure why we would be using different fonts...

Using Futura:
![image](https://user-images.githubusercontent.com/920269/33406104-516b5d98-d539-11e7-90c1-a95c802bf1c3.png)

Using Lato:
![image](https://user-images.githubusercontent.com/920269/33406113-5db25016-d539-11e7-98c2-1cc6848947ea.png)

**Comment by ZacharyDavidSaunders** [comment]
That's interesting. I'm running the same version of Chrome on MacOS High Sierra. We should probably set Futura as our default because windows is more popular than MacOS.

**Comment by bamhm182** [comment]
When I get a moment, I will see if High Sierra shows that don't for me.

**Comment by bamhm182** [comment]
Confirmed that the font is Lato on OS X High Sierra and Futura on Windows 10 for me. Maybe there's some simple check we could implement to show the correct font.

**Comment by ZacharyDavidSaunders** [comment]
Okay, we'll I'm glad that it's at least consistent. I'll try to play with it and see what I can come up with. I know that we can easily get the OS type if we were using node.js, but I'm not familiar with doing it outside of node.
